src/graphics/C4Shader.cpp

```diff
@@ -265,9 +265,9 @@ int C4Shader::ParsePosition(const char *szWhat, const char **ppPos)
 
 	// Lookup name
 	int iPosition = -1;
-	for (unsigned int i = 0; i < sizeof(C4SH_PosNames) / sizeof(*C4SH_PosNames); i++) {
-		if (SEqual(Name.getData(), C4SH_PosNames[i].Name)) {
-			iPosition = C4SH_PosNames[i].Position;
+	for (auto & C4SH_PosName : C4SH_PosNames) {
+		if (SEqual(Name.getData(), C4SH_PosName.Name)) {
+			iPosition = C4SH_PosName.Position;
```

Comment:
The worst thing about the variable naming is that it's not consistent at all, with Hungarian, capitals, camel case and snake case all over the place. The [style guide](http://wiki.openclonk.org/w/Style_Guidelines) mandates a specific style, but it often feels odd to follow that when all the surrounding code is in a different style.